internal/pkg/service/buffer/store/model/column/column.go

```diff
@@ -213,10 +217,85 @@ func (Header) CsvValue(importCtx ImportCtx) (string, error) {
 	return string(header), nil
 }
 
-func (Template) CsvValue(_ ImportCtx) (string, error) {
+func (t Template) CsvValue(importCtx ImportCtx) (string, error) {
+	if t.Language == TemplateLanguageJsonnet {
+		ctx := jsonnet.NewContext()
+		ctx.NativeFunctionWithAlias(getNestedBody("Body", t, importCtx.Body))
+
+		headers := orderedmap.New()
+		for k := range importCtx.Header {
+			headers.Set(k, importCtx.Header.Get(k))
+		}
+		ctx.NativeFunctionWithAlias(getHeader("Headers", t, headers))
+
+		ctx.GlobalBinding("currentDatetime", jsonnet.ValueToLiteral(importCtx.DateTime.Format(time.RFC3339)))
+
+		res, err := jsonnet.Evaluate(t.Content, ctx)
+		if err != nil {
+			return "", err
+		}
+		if res == "null\n" {
+			return "", nil
+		}
+		return strings.TrimRight(res, "\n"), nil
+	}
 	return "", nil
 }
 
+func getNestedBody(name string, t Template, om *orderedmap.OrderedMap) *jsonnet.NativeFunction {
+	return &jsonnet.NativeFunction{
+		Name:   name,
+		Params: ast.Identifiers{"path"},
+		Func: func(params []interface{}) (any, error) {
+			if len(params) != 1 {
+				return nil, errors.Errorf("one parameter expected, found %d", len(params))
+			} else if path, ok := params[0].(string); !ok {
+				return nil, errors.New("parameter must be a string")
+			} else {
+				val, found, err := om.GetNested(path)
+				if !found {
+					if t.UndefinedValueStrategy == UndefinedValueStrategyNull {
+						return nil, nil
+					} else {
+						return nil, errors.Errorf(`Path "%s" not found in the body'`, path)
+					}
+				}
+				if err != nil {
+					return nil, err
+				}
+				if v, ok := val.(*orderedmap.OrderedMap); ok {
+					return v.ToMap(), nil
+				}
+				return val, nil
+			}
+		},
+	}
+}
+
+func getHeader(name string, t Template, om *orderedmap.OrderedMap) *jsonnet.NativeFunction {
+	return &jsonnet.NativeFunction{
+		Name:   name,
+		Params: ast.Identifiers{"path"},
+		Func: func(params []interface{}) (any, error) {
+			if len(params) != 1 {
+				return nil, errors.Errorf("one parameter expected, found %d", len(params))
+			} else if key, ok := params[0].(string); !ok {
+				return nil, errors.New("parameter must be a string")
+			} else {
+				val, found := om.Get(http.CanonicalHeaderKey(key))
+				if !found {
+					if t.UndefinedValueStrategy == UndefinedValueStrategyNull {
+						return nil, nil
+					} else {
+						return nil, errors.Errorf(`Header "%s" not found'`, http.CanonicalHeaderKey(key))
```

Comment:
tu tiez, chyba nema zacinat velkym pismenom, ... na vety to prevadzame na urovni API